### PR TITLE
04, 05, 06 finished ale

### PR DESCRIPTION
**Pull request by DonStella**

### CODE REVIEW

File: 06-PB-ternary-exercises/06ale.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+console.log('question 1');
```
Comment (martl) [inline 06-PB-ternary-exercises/06ale.js:1]: "use strict"

File: 04-PB-operators-arithmetic-pt2/04ale.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+let tiX = 15;
+console.log(6450/tiX);
+//430
+let inCome = 500;
+console.log(inCome * 52.14);
+//26,070
+let percent = 17;
+let total = 30;
+console.log(percent/30 *100);
```
[the remark this answers is not in the document]
Comment (DonStella) [reply 04-PB-operators-arithmetic-pt2/04ale.js:9]: i dont understand what you mean? where do i change?

File: 04-PB-operators-arithmetic-pt2/04ale.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+//56.666
+let side = 4.75;
+console.log(side*4);
+//19
+let triOne = 5;
+let triTwo = 6; 
+let triThree = 7;
+console.log(triOne+triTwo+triThree);
```
[the remark this answers is not in the document]
Comment (DonStella) [reply 04-PB-operators-arithmetic-pt2/04ale.js:17]: ok

File: 04-PB-operators-arithmetic-pt2/04ale.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+//18
+let cube = 9;
+console.log(cube*cube*cube);
+//729
+let result = 17/30 * 100;
+console.log(result);
+//
```
[the remark this answers is not in the document]
Comment (DonStella) [reply 04-PB-operators-arithmetic-pt2/04ale.js:24]: i was just testing something and forgot to delete

File: 04-PB-operators-arithmetic-pt2/04ale.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+let bills = 22.35 + (10/100 * 22.35) + 26.67 + (15/100 * 26.67) + 35.92 + (20/100 * 35.92);
+console.log(bills);
+//98.3595
```
[the remark this answers is not in the document]
Comment (DonStella) [reply 04-PB-operators-arithmetic-pt2/04ale.js:27]: ok

File: 05-PB-operators-comparison/05ale.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+//false
+console.log('Question 9');
+console.log((x/z) == (y/z));
+//false
+console.log('Question 10');
+console.log((z+x) > (y-z));
+//false
+console.log((z+x) == (y-z));
+//true
+
```
[the remark this answers is not in the document]
Comment (DonStella) [reply 05-PB-operators-comparison/05ale.js:38]: this is strict way of writing? and we have to use this way bc it's best practice or why?

File: 06-PB-ternary-exercises/06ale.js (hunk 1 above)
Comment (DonStella) [reply 06-PB-ternary-exercises/06ale.js:1]: ok

**Pull request closed without merging**